app/outliers.py

```diff
@@ -285,6 +285,10 @@ def perform_analysis(housekeeping_job):
             analyzer.unknown_error_analysis = True
             logging.logger.error("error while analyzing use case", exc_info=True)
 
+    try:
```

Comment:
Can you move this flush up in the for loop? (I.e. flush after every model, not only at the end)